src/pathfinding_dijikstra.cpp

```diff
@@ -0,0 +1,414 @@
+#include "pathfinding_dijikstra.h"
+
+#include <algorithm>
+
+#include "cursesdef.h"
+#include "input.h"
+#include "output.h"
+#include "ui_manager.h"
+
+#include "game.h"
+#include "map.h"
+#include "submap.h"
+#include "trap.h"
+#include "veh_type.h"
+#include "vehicle.h"
+#include "vehicle_part.h"
+#include "vpart_position.h"
+
+std::optional<std::vector<tripoint>> DijikstraPathfinding::get_route( const tripoint from,
+                                  const RouteSettings route_settings )
+{
+    if( !g->m.inbounds( from ) ) {
```

Comment:
```suggestion
    if( !get_map().inbounds( from ) ) {
```

`g` brings in whole `game.h` as dependency, so `get_*` variants are preferred